.github/workflows/unit-test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           # make sure the coverage data is cleaned
           rm -rf .nyc_output
           xvfb-run -a npx lerna run test:unit --stream
-          TEAMSFX_INSIDER_PREVIEW=true xvfb-run -a npx lerna run test:unit --stream
+          __TEAMSFX_INSIDER_PREVIEW=false xvfb-run -a npx lerna run test:unit --stream
```

Comment:
It seems some parts are missing for cicd feature.
Can you help remove the non-insider version of cicd support in this PR too? Thanks.
1. remove folder [/docs/cicd](https://github.com/OfficeDev/TeamsFx/tree/dev/docs/cicd)
2. remove non-insider `cicd guide` button, refer to code: [L505](https://github.com/OfficeDev/TeamsFx/blob/15f361caa3c88439dc4c9c324090baf01f27a189/packages/vscode-extension/src/handlers.ts#L505)
3. update readme.md refer to [readme.md](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/cicd_insider/README.md)
![image](https://user-images.githubusercontent.com/1803943/142395599-ed970fd9-bc00-452c-8f8f-2dd9a1755f8d.png)

Reply:
Thanks, I will update it.

Reply:
Offline discussed. 1 and 2 are code cleanup work, and it will be in the next sprint. 3 is updated.